exonum-java-binding-core/rust/integration_tests/tests/utils_errors.rs

```diff
@@ -3,12 +3,12 @@ extern crate java_bindings;
 #[macro_use]
 extern crate lazy_static;
 
-use integration_tests::vm::create_vm_for_tests;
+use integration_tests::vm::create_vm_for_tests_with_fake_classes;
```

Comment:
I switched to this because with just `create_vm_for_tests` it was unable to find `TransactionExecutioException` during initial caching.

Reply:
So are we using `create_vm_for_tests` now?

Reply:
No, for the moment we're using `create_vm_for_tests_with_fake_classes`.

Reply:
I think what Ilya asks is if that method is used at all now.

Reply:
Yes, for the moment we're using it in three places of integrations tests when testing our executors.